.github/workflows/rebase-pull-requests.yml

```
@@ -0,0 +1,9 @@
+name: Rebase Pull Requests
+on:
+  push:
+    branches: [main]
+jobs:
+  rebase:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: linhbn123/rebase-pull-requests@v1.0.1
```

Looked into this in more detail and the GitHub action you're using seems unmaintained.

How about we switch to this, and also add a tag that prevents a PR from being rebased (so we can stop very old ones from being rebased).

https://github.com/peter-evans/rebase

```suggestion
name: Rebase
on:
  push:
    branches: [main]
jobs:
  rebase:
    runs-on: ubuntu-latest
    steps:
      - uses: peter-evans/rebase@v2
        with:
          base: main
          exclude-labels: |
            no-rebase

```